src/App.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { Link, Route, Routes } from 'react-router-dom';
+import './styles/style.scss';
+import Books from './combiner/Books';
+import Cartegory from './components/Category';
+
+export default function App() {
+  return (
+    <>
+      <nav className="navbar">
+        <div className="Title-name">
+          BOOK STORE
+        </div>
+        <div className="nav-links">
+          <Link to="/Books"> Books </Link>
+          <Link to="/Category"> Category </Link>
+        </div>
+      </nav>
+      <Routes>
+        <Route path="/Category" element={<Cartegory />} />
+        <Route path="/Books" element={<Books />} />
+        <Route path="/" />
```

Comment:
- [ ] In the route where you are using the Books component, kindly have the path to just "/" so the homepage route renders the books. Currently, you are having an empty route with "/" that is showing an empty page that does not exist. The goal is to make the "/" route the homepage route and shows all the books. Kindly work on that 👍🏽

Reply:
Change the path as requested.

Reply:
Change the links path